src/Apps/Collect2/Routes/Collection/index.tsx

```diff
@@ -36,8 +36,8 @@ interface CollectionAppProps extends SystemContextProps {
 
 @track<CollectionAppProps>(props => ({
   context_module: Schema.ContextModule.CollectionDescription,
-  context_page_owner_slug: props.viewer.slug,
-  context_page_owner_id: props.viewer.id,
+  context_page_owner_slug: props.viewer && props.viewer.slug,
```

Comment:
Kinda funny, `viewer` would be `null` for collections and so would crash on the tracking decorator. Now this correctly falls thru to `collectionNotFound` and throws a 404.